.circleci/config.yml

```diff
@@ -38,10 +38,12 @@ jobs:
             opam pin add -y -n sihl_user_mariadb .
             opam pin add -y -n sihl_user_postgresql .
       - run:
+          # Remove jwto once 0.3.0 is in OPAM
```

Comment:
Can you add TODO tag to this comment so that it is not lost

Reply:
Done